src/handlers/TechRecordsListHandler.ts

```diff
@@ -17,7 +17,8 @@ export class TechRecordsListHandler<T extends Vehicle> {
   public async getFormattedTechRecordsList(
     searchTerm: string,
     status: string,
-    searchCriteria: ISearchCriteria = SEARCHCRITERIA.ALL
+    searchCriteria: ISearchCriteria = SEARCHCRITERIA.ALL,
+    mergeRecordsWithSameSystemNumber: boolean = true
```

Comment:
Why would you need to conditionally merge them?

Reply:
Most endpoints use this function to get what's in Dynamo. It's a lot easier internally for endpoints to be aware of the actual data in Dynamo rather than misrepresenting what's in Dynamo for those endpoints. 

E.g. here there's a function that makes a request to Dynamo to update a record. This function can't tell Dynamo which record to update if it doesn't know what shape the data actually has in Dynamo